### PR TITLE
ktlint: Add version 1.0.0 (Close #5076)

### LINKED ISSUE #5076: [Request]: ktlint
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

ktlint

### Description

Kotlin linter in spirit of feross/standard (JavaScript) and gofmt (Go).

### Homepage

https://pinterest.github.io/ktlint/

### Download Link(s)

https://github.com/pinterest/ktlint/releases

### Some Indication of Popularity/Repute

1. There is an [App manifest](https://github.com/ScoopInstaller/Extras/blob/master/bucket/ktlint.json) in Extras bucket, URLs in the manifest are old though …
2. GitHub 5.7k Stars
3. It is a command line tool!

### PR DESCRIPTION
**Pull request by Elkiwa**
Closes #5076

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/ktlint.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "url": "https://github.com/pinterest/ktlint/releases/download/1.0.0/ktlint#/ktlint.jar",
+    "hash": "5d19ef51d198eecf1280fb167192ed1e1374c70eb9311abceb89207ad06481dd",
+    "bin": "ktlint.jar",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/pinterest/ktlint/releases/download/$version/ktlint#/ktlint.jar"
```
Comment (rashil2000) [inline bucket/ktlint.json:17]:
```suggestion
        "url": "https://github.com/pinterest/ktlint/releases/download/$version/ktlint#/ktlint.jar",
        "hash": {
            "url": "$url.md5"
        }
```

**Comment by rashil2000** [comment]
ktlint is already present in the Extras bucket: https://github.com/ScoopInstaller/Extras/blob/master/bucket/ktlint.json. 

Please raise a PR in that bucket (and tag me) to remove it from there (to prevent duplication).

File: bucket/ktlint.json (hunk 1 above)
Comment (Elkiwa) [reply bucket/ktlint.json:17]: Thanks for your review. However,
1. [ktlint@0.48.2](https://github.com/pinterest/ktlint/releases/tag/0.48.2) seems to be the last release utilizing `md5` hashes, but not anymore.
image

2. They only provide a PGP signature (`ktlint.asc` contains PGP signature) mentioned in their [documentation](https://pinterest.github.io/ktlint/1.0.0/install/cli/). 

Would you still like to add this section or other modifications?
```        
"hash": {
            "url": "$url.md5"
 }
```

**Pull request merged**